### DISCUSSION TITLE
GET https://registry.npmjs.org/next failes or takes very very long

### DISCUSSION THREAD

**Discussion by florianmutter** [npm/feedback#765] [Registry]
Currently our dependency update is failing because the GET request to https://registry.npmjs.org/next takes to long. When running it in Chrome it takes from 2 Minutes to not finishing at all.

![grafik](https://user-images.githubusercontent.com/32459530/189050882-9318d80b-d9c8-494c-840b-d1ebbf2e80cf.png)

It seems that the file size is 45MB. This seems big but should not result in such a slow download.

**Comment by juanpicado** [comment]
I think the responsibility here is not only the public registry, also its users, in this specific case is how Next.js https://www.npmjs.com/package/next is publishing snapshots versions on the registry Next has published 213 versions this year (we are still in September) and only 18 are final releases (without canary), the same pattern for other libraries, unfortunately, https://www.npmjs.com/package/typescript, and that rhythm with the same system is gonna be unsustainable in few years download more popular package decides to follow the same pattern. 

I think a discussion about this is needed.

For every published version a copy of the metadata is added to the _versions_ fields, for canaries means 99% duplication + the field _time_ field which is much easier to notice a big part of the problem. 

And for some reason, some versions (I haven't checked all) include the readme which is quite a bunch of text.

<img width="537" alt="Screenshot 2022-09-08 at 14 04 38" src="https://user-images.githubusercontent.com/558752/189117368-50bfcdb5-a4a1-4f1d-84e7-130dd3f6765d.png">
